subsys/net/ip/net_context.c

```diff
@@ -368,23 +368,19 @@ static int check_used_port(enum net_ip_protocol ip_proto,
 static u16_t find_available_port(struct net_context *context,
```

Comment:
To my eyes it looks like there is a typo in commit message:

s/This will leads a problem if the port number is already used./
This will lead to problems if the port number is already used.